policies/intel-aws-db-instance-deny-unapproved-instance-types/intel-aws-db-instance-deny-unapproved-instance-types.sentinel

```diff
@@ -0,0 +1,127 @@
+import "tfplan/v2" as tfplan
+
+param valid_actions default [
+	["no-op"],
+	["create"],
+	["update"],
+]
+
+format = {
+	"reset": "\033[0m",
+	"info":  "\033[32m",
+	"warn":  "\033[33m",
+	"error": "\033[31m",
+}
+
+summary = func(input, level) {
+	result = false
+	if input.violations is empty {
+		return true
+	} else {
+		log(input, format[level])
+	}
+	return result
+}
+
+log = func(input, output) {
+	header(input, output)
+	violations(input, output)
+	return null
+}
+
+header = func(input, output) {
+	print(
```

Comment:
We should look at moving these into a shared function library that can be imported and take inputs for generating the required objects for the output report.